Parsers/ASimWebSession/Parsers/ASimWebSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,341 @@
+Parser:

```

Comment:
@jaimeesc I reviewed the network parser PR. Please apply the changes as suggested in the other PR.
Please keep only fields that are relevant to WebSession schema here:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-web#schema-overview

Please filter out logs if URL not present as its mandatory.